### PR TITLE
chore: add params for wescale

### PR DESCRIPTION
**Pull request by earayu**
1. Refactor bootstrap script for wescale
2. add 'ddl_strategy' param for wescale VTGate
3. Let wescale use 'autoTrigger' in ConfigConstraint
4. upgrade wescale version from 0.2.3 to 0.2.4

### CODE REVIEW

**Comment by earayu** [comment]
Could you kindly take a moment to review this PR for me, please? Your insights would be greatly appreciated.
@heng4fun @shanshanying

File: addons/apecloud-mysql/config/mysql-scale-vtgate-config-constraint.cue [hunk 1]
```diff
@@ -83,6 +83,9 @@
 	//Reject insecure connections but only if mysql_server_ssl_cert and mysql_server_ssl_key are provided.(default "false")
 	mysql_server_require_secure_transport: bool
 
+	// Set default strategy for DDL statements. Override with @@ddl_strategy session variable
+	ddl_strategy: string & "direct" | "online" | "mysql"
```
Comment (heng4fun) [inline addons/apecloud-mysql/config/mysql-scale-vtgate-config-constraint.cue:87]: What's the different between `direct` and `mysql`?

Comment (earayu) [reply addons/apecloud-mysql/config/mysql-scale-vtgate-config-constraint.cue:87]: Direct: Direct forwarding.
MySQL: Includes a task management mechanism, such as the capability to delay the start of DDL operations.
Online:  Includes a task management mechanism, in addition to the capability for Online DDL.

**Review by heng4fun — APPROVED** [review]
LGTM

**Pull request merged**